File: task1.py
# Write a program to reverse an array or string



def reverseList (A, start, end):
    while start < end:
        A[start], A[end] = A[end],A[start]
        start += 1
        end -= 1
    
    return A

    # reversed =""
    # # print(A)
    # for edx in range(len(A)-1 , -1, -1 ):
    #     # print(A[edx])
    #     reversed += str(A[edx])
    # return reversed


A = [11,22,33,44]
# A = "I am new to python"
print (reverseList(A, 0, 3))

File: task1.js
// Write a program to reverse an array or string



function reverseString(string) {
    //     let str = "ABC"
    // lenght 3 
    // index 2 
    let reversedString = ""
    for (let i = string.length - 1; i >= 0; i--) {
        reversedString += string[i];
    }

    return reversedString

}


let string = [1, 2, 3]


console.log("Your Output ", reverseString(string))
